VTEX - Orders API.json

```diff
@@ -5712,7 +5712,3709 @@
+            "post": {
+              "tags": [
+                "Change order"
+              ],
+              "summary": "Cancel order change",
+              "operationId": "CancelChangeStoppedonError",
+              "description": "This endpoint cancels an order change. It is meant for when, due to an error, a change was not completely processed, and you do not wish to retry it. For this request, you need the order's `changeRequestId`, the unique identification code generated by the Order change endpoint.\r\n\r\nOnce an order change has been successfully processed, canceling it is no longer possible. If you wish to revert the changes made to an order, you will have to create a new order change.\r\n\r\n## Permissions\r\n\r\nAny user or [application key](https:\/\/developers.vtex.com\/docs\/guides\/authentication-overview#application-keys) must have at least one of the appropriate [License Manager resources](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3) to be able to successfully run this request. Otherwise they will receive a status code `403` error. These are the applicable resources for this endpoint:\r\n\r\n| **Product** | **Category** | **Resource** |\r\n| --------------- | ----------------- | ----------------- |\r\n| Checkout | CheckoutResources | **Orders Full Access** |\r\n| OMS | OMS access | **Cancel order** |\r\n| OMS | OMS access | **Change order** |\r\n\r\nYou can [create a custom role](https:\/\/help.vtex.com\/en\/tutorial\/roles--7HKK5Uau2H6wxE1rH5oRbc#creating-a-role) with that resource or use one of the following [predefined roles](https:\/\/help.vtex.com\/en\/tutorial\/predefined-roles--jGDurZKJHvHJS13LnO7Dy):\r\n\r\n| **Role** | **Resource** | \r\n| --------------- | ----------------- | \r\n| Checkout Admin | Orders Full Access |\r\n| OMS - Full access | <p>Cancel order<\/p><p>Change order<\/p> |\r\n| IntegrationProfile - Fulfillment Oms | <p>Cancel order<\/p><p>Change order<\/p> |\r\n| IntegrationProfile - Fulfillment Gateway | Cancel order |\r\n| IntegrationProfile - Fulfillment Gateway Oms | <p>Cancel order<\/p><p>Change order<\/p> |\r\n| inStore Sales Person | <p>Cancel order<\/p><p>Change order<\/p> |\r\n\r\n>\u2757 Assigning a [predefined role](https:\/\/help.vtex.com\/en\/tutorial\/predefined-roles--jGDurZKJHvHJS13LnO7Dy) to users or application keys usually grants permission to multiple [License Manager resources](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3). If some of these permissions are not necessary, consider creating a custom role instead. For more information regarding security, see [Best practices for using application keys](https:\/\/help.vtex.com\/en\/tutorial\/best-practices-application-keys--7b6nD1VMHa49aI5brlOvJm).\r\n\r\nTo learn more about machine authentication at VTEX, see [Authentication overview](https:\/\/developers.vtex.com\/docs\/guides\/authentication-overview#machine-authentication).",
```

Comment:
> This endpoint cancels an order change. It is meant for when, due to an error, a change was not completely processed, and you do not wish to retry it. For this request, you need the order's changeRequestId, the unique identification code generated by the Order change endpoint.

This part would be greatly improved if you could add this info:
* Where do you get this ID? What request? What response status code? Probably 4xx or 5xx something, but it really depends on how the developers made it.
* What is the key under which you get this ID? Many APIs just return an id field or something like that instead of a more precise name.

Reply:
- After approving this PR, I will add the enpoints' links and the text will make sense because it says ", the unique identification code generated by the Order change endpoint." The key is `changeRequestId`.

- About the errors, the team also said it could really be any type of error, just like that other endpoint where you made this consideration. I will get back to that point with them again, but meanwhile I will not keep this PR on hold.